cfgov/cfgov/settings/base.py

```diff
@@ -570,6 +570,9 @@
                    'bam.nr-data.net',
                    'files.consumerfinance.gov',
                    's3.amazonaws.com',
+                   # Todo: Take the following URL out as it's only
+                   # for http access to chart data for CCT
```

Comment:
Let's get slightly more specific about when this can be removed (when Build runs on HTTPS).

Reply:
It can be removed either when build runs on HTTPS or when the cfpb-chart-builder PR is merged. I'll update.

Reply:
Do we know for sure that accessing an HTTPS resource from an HTTP page is a problem? It seems like it shouldn't be.

(the opposite definitely IS a problem, of course)